octobot/octobot.pxd

```diff
@@ -34,6 +34,7 @@ cdef class OctoBot:
 
     cdef public dict tools
     cdef public dict config
+    cdef public str id
```

Comment:
I don't think `id` is a good idea, it's a [reserved python keyword](https://docs.python.org/3.5/library/functions.html#id)

Reply:
right